cap/event_exports.go

```diff
@@ -48,3 +51,87 @@ type Event = s.Event
 //
 // Since: 0.0.0
 type EventNotifier = s.EventNotifier
+
+// ReliableNotifierOpt allows clients to tweak the behavior of an EventNotifier
+// instance built with NewReliableNotifier
+//
+// Since: 0.1.0
+type ReliableNotifierOpt = n.ReliableNotifierOpt
+
+// NewReliableNotifier is an EventNotifier that guarantees it will never panic
+// the execution of its caller, and that it will continue sending events to
+// notifiers despite previous panics
+//
+// Since: 0.1.0
+var NewReliableNotifier = n.NewReliableNotifier
+
+// WithOnNotifierTimeout sets callback that gets executed when a given notifier drops
+// an event because it cannot process it.
+//
+// Since: 0.1.0
+var WithOnNotifierTimeout = n.WithOnNotifierTimeout
+
+// WithNotifierTimeout sets the maximum allowed time the reliable notifier is going to
+// wait for a notifier function to be ready to receive an event (defaults to 10 millis).
+//
+// since: 0.1.0
+var WithNotifierTimeout = n.WithNotifierTimeout
+
+// WithOnReliableNotifierFailure sets a callback that gets executed when a failure
+// occurs on the event broadcasting logic
+//
+// Since: 0.1.0
+var WithOnReliableNotifierFailure = n.WithOnReliableNotifierFailure
```

Comment:
This is correct; this reports failures when we notify sentry, or logs, or metrics. If we happen to notify some service that has a flaky behaviour, this will probably get called if it fails more than the restart threshold